package.json

```diff
@@ -97,6 +98,7 @@
     "mailparser": "0.6.1",
     "mkdirp": "0.5.1",
     "mocha": "3.1.2",
+    "mustache": "2.3.0",
```

Comment:
we have `handlebars` in deps already. Can we just use that? 

I'm not a fan of mustache, IMHO is limiting and leads to spaghetti templates.